sal_interfaces/SALSubsystems.xml

```diff
@@ -271,87 +271,87 @@
-  <IndexEnumeration></IndexEnumeration>
+  <IndexEnumeration>no</IndexEnumeration>
   <Generics>yes</Generics>
   <ActiveDevelopers>Russell Owen</ActiveDevelopers>
   <Github>https://github.com/lsst-ts/ts_ATDome</Github>
-  <JenkinsTestResults>NA</JenkinsTestResults>
+  <JenkinsTestResults>Not Available</JenkinsTestResults>
   <RubinObsContact>Patrick Ingraham</RubinObsContact>
   <CSCDocs>https://ts-atdome.lsst.io/</CSCDocs>
   <ProductOwner>Patrick Ingraham</ProductOwner>
   <RelatedDocuments>Manual Document-27878</RelatedDocuments>
   <SoftwareLanguage>Python</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>The control system consists of an in house CSC that talks over TCP/IP to a low-level controller written by Astronomical Consulting Equipment.</VendorContact>
-  <Simulator>Yes</Simulator>
-  <Configuration>Yes</Configuration>
+  <Simulator>Internal to CSC</Simulator>
+  <Configuration>https://github.com/lsst-ts/ts_config_attcs</Configuration>
 </SALSubsystem>
 
 <SALSubsystem>
   <Name>ATDomeTrajectory</Name>
   <Description>Command the auxiliary telescope dome (ATDome) to follow the telescope.</Description>
-  <IndexEnumeration></IndexEnumeration>
+  <IndexEnumeration>no</IndexEnumeration>
   <Generics>yes</Generics>
   <ActiveDevelopers>Russell Owen</ActiveDevelopers>
   <Github>https://github.com/lsst-ts/ts_ATDomeTrajectory</Github>
-  <JenkinsTestResults></JenkinsTestResults>
+  <JenkinsTestResults>https://tssw-ci.lsst.org/job/ATDomeTrajectory</JenkinsTestResults>
   <RubinObsContact>Patrick Ingraham</RubinObsContact>
   <CSCDocs>https://ts-atdometrajectory.lsst.io/</CSCDocs>
   <ProductOwner>Patrick Ingraham</ProductOwner>
   <RelatedDocuments>NA</RelatedDocuments>
   <SoftwareLanguage>Python</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>NA</VendorContact>
-  <Simulator>No</Simulator>
-  <Configuration>No</Configuration>
+  <Simulator>Not Required</Simulator>
+  <Configuration>https://github.com/lsst-ts/ts_config_attcs</Configuration>
 </SALSubsystem>
 
 <SALSubsystem>
   <Name>ATMCS</Name>
   <Description>The CSC for the Auxiliary Telescope Mount Control System</Description>
-  <IndexEnumeration></IndexEnumeration>
+  <IndexEnumeration>no</IndexEnumeration>
   <Generics>yes</Generics>
   <ActiveDevelopers>Rolando Cantarutti from CTIO</ActiveDevelopers>
   <Github>Not yet available publicly</Github>
-  <JenkinsTestResults>NA</JenkinsTestResults>
+  <JenkinsTestResults>Not </JenkinsTestResults>
   <RubinObsContact>Patrick Ingraham</RubinObsContact>
   <CSCDocs>NA</CSCDocs>
   <ProductOwner>Patrick Ingraham</ProductOwner>
   <RelatedDocuments>No manuals yet but there will be. Still being written by CTIO</RelatedDocuments>
   <SoftwareLanguage>LabVIEW</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>CTIO</VendorContact>
-  <Simulator>Yes</Simulator>
-  <Configuration>No</Configuration>
+  <Simulator>ts_ATMCSSimulator</Simulator>
```

Comment:
I assume this should be the URL to the repo:
https://github.com/lsst-ts/ts_ATMCSSimulator